algorithms/ar-lis/javascript/longest_increasing_subsequence.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Longest increasing subsequence is the subsequence that has the longest streak of going up consecutively
+ * @param {number[]} arr the sequence to search for longest subsequence
+ */
+function findLongestIncreasingSubsequence(arr) {
+    let longestSubsequence = [];
+    for (let arrKey in arr) {
```

Comment:
Codacy found an issue: [The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6422328)